File: tests/human/x/fortran/string_prefix_slice.f90
program string_prefix_slice
  implicit none
  character(len=4) :: prefix
  character(len=6) :: s1, s2
  prefix = 'fore'
  s1 = 'forest'
  print '(A)', trim(merge('true ','false', s1(1:len(prefix)) == prefix))
  s2 = 'desert'
  print '(A)', trim(merge('true ','false', s2(1:len(prefix)) == prefix))
end program string_prefix_slice

File: tests/human/x/cs/string_prefix_slice.cs
using System;

class Program
{
    static void Main()
    {
        string prefix = "fore";
        string s1 = "forest";
        Console.WriteLine(s1.Substring(0, prefix.Length) == prefix);
        string s2 = "desert";
        Console.WriteLine(s2.Substring(0, prefix.Length) == prefix);
    }
}
